test/resources/read-file-clustertask.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: tekton.dev/v1alpha1
 kind: ClusterTask
 metadata:
-  name: read-clustertask
+  generateName: read-clustertask-
```

Comment:
Why do we need to use `generateName`?

Reply:
If the same set of tests are run in multiple platforms in parallel, having the same name for the cluster task causes problem.